### PR TITLE
Add community announcement for CVE-2019-12995

### PR DESCRIPTION
**Pull request by fpesce**
Please provide a description for what this PR is for.

Blog post for CVE-2019-12995 announcement.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/blog/2019/cve-2019-12995/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+if [ "${#JWKS_URI[@]}" != 0 ]; then
+  echo "${red}Found JWT in authentication policy, YOU ARE AFFECTED${reset}"
+  exit 1
+fi
+
+echo "${green}Did NOT find JWT in authentication policy, YOU ARE NOT AFFECTED${reset}"
+EOF
+```
+
+## Mitigation
```
Comment (geeknoid) [inline content/blog/2019/cve-2019-12995/index.md]: Plural

File: content/blog/2019/cve-2019-12995/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+A bug in Istio’s JWT validation filter causes Istio Proxy to crash in certain cases when the request contains a malformed JWT token. The bug was discovered and reported by a user [on github](https://github.com/istio/istio/issues/15084) on June 23, 2019.
```
Comment (geeknoid) [inline content/blog/2019/cve-2019-12995/index.md]: Needs front-matter.
Uses of triple-ticks must be replaced with `{{<text>}}`

File: content/blog/2019/cve-2019-12995/index.md (hunk 1 above)
Comment (fpesce) [reply content/blog/2019/cve-2019-12995/index.md]:
```suggestion
## Mitigations
```

File: content/blog/2019/cve-2019-12995/index.md (hunk 2 above)
Comment (fpesce) [reply content/blog/2019/cve-2019-12995/index.md]: Not sure to understand, would you mind provide a suggestion?

File: content/blog/2019/cve-2019-12995/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+If JWT policy is applied to the istio ingress gateway, please be aware that any external user who has access to the ingress gateway could crash it with a single HTTP request.
+
+If JWT policy is applied to the sidecar only, please keep in mind it might still be vulnerable. For example, the istio ingress gateway might forward the JWT token to the sidecar which could be a malformed JWT token that crashes the sidecar.
+
+A vulnerable Istio proxy will crash on an HTTP request with a malformed JWT token. When Istio proxy crashes, all existing connections will be disconnected immediately. The pilot-agent will restart the crashed Istio proxy automatically and it may take a few seconds to a few minutes for the restart. pilot-agent will stop restarting the Istio proxy after it crashed more than 10 times. In this case, Kubernetes will redeploy the pod (including the workload behind Istio proxy).
+
+To detect if there is any JWT authentication policy applied in your cluster, run the following command which print either of the following output:
+* Found JWT in authentication policy, **YOU ARE AFFECTED**
+* Did NOT find JWT in authentication policy, *YOU ARE NOT AFFECTED*
+```bash
```
Comment (fpesce) [inline content/blog/2019/cve-2019-12995/index.md]:
```suggestion
{{< text bash >}}
```

File: content/blog/2019/cve-2019-12995/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+JWKS_URI+=($(kubectl get policy --all-namespaces -o jsonpath='{range .items[*]}{.spec.origins[*].jwt.jwksUri}{" "}{end}'))
+JWKS_URI+=($(kubectl get meshpolicy --all-namespaces -o jsonpath='{range .items[*]}{.spec.origins[*].jwt.jwksUri}{" "}{end}'))
+if [ "${#JWKS_URI[@]}" != 0 ]; then
+  echo "${red}Found JWT in authentication policy, YOU ARE AFFECTED${reset}"
+  exit 1
+fi
+
+echo "${green}Did NOT find JWT in authentication policy, YOU ARE NOT AFFECTED${reset}"
+EOF
+```
```
Comment (fpesce) [inline content/blog/2019/cve-2019-12995/index.md]:
```suggestion
{{< /text >}}
```

**Comment by fpesce** [comment]
Circle CI error does not look to happen in my changes, @geeknoid PTAL

**Pull request merged**